extensions/windows-event-log/tests/CWELTestUtils.h

```diff
@@ -17,6 +17,10 @@
 
 #pragma once
 
+#include <utility>
+#include <memory>
+#include <string>
```

Comment:
Missing `<fstream>` (`std::ifstream` at line 80) and `<iterator>` (`std::istreambuf_iterator` at line 81).

Reply:
thanks, fixed in [95aeacd](https://github.com/martinzink/nifi-minifi-cpp/commit/95aeacd78262361de87609058f1f208b01e4fe3b)